### ISSUE TITLE
Automatic reviewer assignation based on the labels applied to the issue that resolves the PR

### ISSUE THREAD

**Issue by maruiz93** [kubearchive/kubearchive#550]
Based on the owners of each area/component we would want to do an automatic assignation of reviewers based on the labels of the issue.

For that, we need to:
* Copy the issue labels to the linked PR -> Use https://github.com/marketplace/actions/copy-issue-labels
* Assign reviewers based on the owners of each area/component -> Use https://github.com/marketplace/actions/assign-reviewers-by-labels

**Comment by rh-hemartin** [comment]
Couldn't automatic assignmed by folder work for us? Its provided by default by GH. https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

**Comment by maruiz93** [comment]
> Couldn't automatic assignmed by folder work for us?

I don't think so, or at least not if we want to use "owners" per area and not just per "component". I was thinking of using an owner for performance, an owner for database, an owner for metrics... and those things are kind of cross-cutting through different pieces of code and folders.
